Sources/TuistCore/Models/Settings.swift

```diff
@@ -32,7 +32,7 @@ public enum SettingValue: ExpressibleByStringLiteral, ExpressibleByArrayLiteral,
 public struct Configuration: Equatable {
     // MARK: - Attributes
 
-    public let settings: SettingsDictionary
+    public var settings: SettingsDictionary
```

Comment:
Same here with mutations. I'd expose a method that gives you a copy of `Configuration` with a given settings:
```
config.with(settings: newSettings)
```